lib/cloudinary/utils.rb

```diff
@@ -113,6 +113,8 @@ class Cloudinary::Utils
       zoom
   ].map(&:to_sym)
 
+  REMOTE_URL_REGEX = %r(^ftp:|^https?:|^s3:|^gs:|^data:([\w-]+\/[\w-]+)?(;[\w-]+=[\w-]+)*;base64,([a-zA-Z0-9\/+\n=]+)$)
```

Comment:
with multiline strings, it is better to use `\A` and `\z` instead of `^` and `$`.
for example, `"abc\nxyz\nabc" =~ /^xyz$/` holds